src/Sylius/Behat/Context/Api/Admin/ManagingChannelsContext.php

```diff
@@ -111,10 +109,9 @@ public function iSpecifyItsAs(string $field, string $value = ''): void
      */
     public function iChooseAsTheBaseCurrency(?CurrencyInterface $currency = null): void
     {
-        $this->client->addRequestData(
-            'baseCurrency',
-            null === $currency ? $currency : $this->sectionAwareIriConverter->getIriFromResourceInSection($currency, 'admin'),
-        );
+        if ($currency !== null) {
+            $this->client->addRequestData('baseCurrency', $this->iriConverter->getIriFromResource($currency));
+        }
```

Comment:
Passing a `null` fails on IRI conversion. We should investigate whether `PATCH` is still the default way of updating resources in APIP3.